### DISCUSSION TITLE
4k时光 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#128] [模板讨论]
4k时光 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by icedog111** [comment]
大佬不能用了，看看应该怎么修改

Failed at 1/4 request,Fail assert: {"re": "200", "from": "status"} from success_asserts,Response Error : HTTP 302: Found,Request URL: https://www.4ksg.com/plugin.php?id=dc_signin

**Reply by wjf0214** [reply]
昨天不就更了。

**Reply by wjf0214** [reply]
更新了.

**Reply by icedog111** [reply]
在佬，还是不行，
2023-09-15 16:17:42
日志：Failed at 1/8 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 302: Found,
Request URL: https://www.4ksg.com/plugin.php?id=dc_signin

**Reply by wjf0214** [reply]
更新到最新的看看。框架有缓存的话，从github上下载试试。

**Reply by icedog111** [reply]
好的

**Comment by miaoxiaozhao** [comment]
签到信息：【if(typeof succeedhandle_signin=='function') {succeedhandle_signin('https://www.4ksg.com/plugin.php?id=dc_signin', '签到成功~', {});}】
时光币：【256】，威望：【0】，贡献：【0】，积分：【256】

可以优化一下显示

**Reply by wjf0214** [reply]
先凑合吧。首次和第二次请求返回结果不一样。设置自动时间早，懒得看返回信息了。不影响签到。

**Comment by llemllen** [comment]
现在签到不了,提示
HTTP 599: Recv failure: Connection reset by peer,
